runner/src/rpc/api.rs

```diff
@@ -230,68 +228,8 @@ pub async fn dev_create() -> Result<(AccountId, InMemorySigner), String> {
 
 pub async fn dev_deploy(contract_file: &Path) -> Result<(AccountId, InMemorySigner), String> {
```

Comment:
Don't necessarily need to worry about generic usage for this so can use `impl AsRef<Path>` to allow this to be called like `dev_deploy("<some-path>")` and it'll remove boilerplate from tests